types/jsx-pdf/index.d.ts

```diff
@@ -0,0 +1,48 @@
+// Type definitions for jsx-pdf 2.2
+// Project: https://github.com/schibsted/jsx-pdf#readme
+// Definitions by: Kyle Hensel <https://github.com/k-yle>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Minimum TypeScript Version: 3.9
+import * as PDFMake from 'pdfmake/interfaces';
+
+export function renderPdf(jsx: JSX.Element): PDFMake.TDocumentDefinitions;
+
+declare global {
+    namespace JSX {
```

Comment:
@orta @DanielRosenwasser  do any of you know this library. I am not sure adding jsx in global is right thing to do or not esp whats the impact when these types are imported with or without react.

Reply:
I don't know this lib directly, but I know the genre. 

I compared this with [the global.d.ts](https://github.com/vadimdemedes/ink/blob/master/src/global.d.ts#L6) in [Ink](https://github.com/vadimdemedes/ink), which imports some of of the types from react directly. I'd _imagine_ this is probably less likely to cause issues, but for both `Element` and `IntrinsicElements` it's using interface merging which should mean it works out of the box with React.

So, I'm a 👍🏻 for the PR, but I think it's worth getting a reply from @k-yle about whether they looked into re-using some of the imports from react

Reply:
Hi @orta, I'm certainly no expert on the matter.. my limited understanding made me think that if `@types/jsx-pdf` declares the global JSX namespace, then importing `@types/react` which does the same would cause problems. 

You linked to [Ink](https://github.com/vadimdemedes/ink/blob/master/src/global.d.ts?rgh-link-date=2020-09-21T21%3A50%3A19Z#L6), which imports `ReactNode`, `Key`, and `LegacyRef` from react. We don't need refs, and `Key` is just `string | number`. Similarly, `ReactNode` is just `string | number | boolean | undefined | null | Element | Element[]`. So I didn't think we actually need anything from react since those are trivial enough for us to define ourselves.

But if it works for Ink and it's simpler, then I can make the change

Reply:
Ah yeah, was just coming back today to say this feels reasonable to me as-is 👍🏻